### PR TITLE
blacken code and add github action to test for compliance.

### PR DESCRIPTION
**Pull request by addyess**
Created by running `black` and testing for any non-compliant black code

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[flake8]
+max-line-length = 88
+
+[tox]
+skipsdist = True
+envlist = lint
+
+[testenv]
+basepython = python3
```
Comment (knkski) [inline tox.ini]: Is this line actually necessary? I haven't run into any issues in a while where `python` was pointing at a `python2` binary.

File: .github/workflows/tox.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Run tests with Tox
+
+on: [push]
```
Comment (knkski) [inline .github/workflows/tox.yaml]: Probably want this to have `pull_request` as well?

**Review by knkski — APPROVED** [review]
Two minor nits but otherwise LGTM

File: tox.ini (hunk 1 above)
Comment (addyess) [reply tox.ini]: lets find out!

**Review by kwmonroe — APPROVED** [review]

**Pull request merged**